src/components/NewWorkspaceModal.js

```diff
@@ -409,6 +410,11 @@ const NewWorkspaceModal = withDisplayName(
                     ]),
                 ]),
               customMessage && div({ style: { marginTop: '1rem', lineHeight: '1.5rem' } }, [customMessage]),
+              disallowAzure &&
+                div({ style: { paddingTop: '1.0rem', display: 'flex' } }, [
+                  icon('warning-standard', { size: 16, style: { marginRight: '0.5rem', color: colors.warning() } }),
+                  div(['New Azure workspace are not currently supported here.']),
```

Comment:
This is grammatically incorrect and also offers no explanation of why it is not supported. Could you work with UX (Kate is our UX person) to get a message that is more useful? I am imagining something like "Only Google workspaces can be created when importing a workflow".

Reply:
Yeah, reframing the input as `workflowImport` makes this make a lot of sense too.

Reply:
I've pinged UX to opine, does this also look better to you?

Reply:
Thanks @cjllanwarne, it looks better, but I'm worried that it might still be a bit confusing because what we are trying to explain is why there are no Azure billing projects in the Billing project select.

Maybe "Importing a workflow into a new workspace backed by an Azure billing project is not currently supported"? But that's certainly a mouthful….

We could also consider only showing the message if we actually filtered out Azure billing projects so that we don't unnecessarily confuse users who care nothing about Azure. And that makes me realize that you also need to check what happens if they user *only* has Azure billing projects. Will they get a dialog that they can do nothing with, or will the get a perhaps similarly confusing dialog that says they need to create a billing project?

Reply:
> Maybe "Importing a workflow into a new workspace backed by an Azure billing project is not currently supported"? But that's certainly a mouthful….

How about "Azure billing projects cannot currently be used for new workspaces during workflow import"?

I haven't heard back from UX, I'll trying pinging Sara instead. I hope that with a good enough message, an empty list is fine and it's still obvious what the reason is for it.

Reply:
I suppose a bigger question is if you only have GCP billing projects, seeing this "warning" message appear might look like something's going wrong, rather than it helping the smaller number of Azure users who might see it

Reply:
Perhaps reframing this as an info message with "To create a new workspace with an Azure billing project, visit the main Workspaces tab" is better?

Reply:
I think we should only show the message if we have filtered out Azure billing projects.

And I think that if we do show it, we should explain that you can't import a workflow into an Azure workspace… going to the main Workspaces tab doesn't help you import a workflow.

Reply:
> we should explain that you can't import a workflow into an Azure workspace… going to the main Workspaces tab doesn't help you import a workflow.

For context, this is exactly the change that's currently in development. So importing into **_existing_** azure workspaces will work, it's just that you won't be able to do everything all at once (create a workspace, AND turn on the workflows app, AND import into it, all in the same operation, while you're sitting on the workflow import page waiting. )

Reply:
Oh wow, that's confusing. I was assuming that import of a workflow into an Azure workspace was the thing that wasn't supported.

Reply:
Update from Katie in UX:

---

From a UX perspective, the language in the mock up is clear enough to help people understand that the functionality is not yet supported. And adding the line that let’s them know what they can do about it would be most helpful. My recommendation would be to use the following message and provide the link to the Workspaces tab:
> "Importing directly into new Azure workspaces is not currently supported. To create a new workspace with an Azure billing project, visit the main Workspaces tab."

- Suggestion: This message could be presented as an informational message, rather than a warning message.
- Lastly, I would agree that the message should only be shown when Azure projects are filtered out. 

--- 

... which sounds good to me. I'll get going on making the changes!